android/AndroidOpenGLESLessons/app/src/main/java/com/learnopengles/android/TableOfContents.java

```diff
@@ -1,137 +1,137 @@
-package com.learnopengles.android;

```

Comment:
Couple of files had windows line endings, CRLF (the rest of project files are in LF) , so the IDE fixed them to be LF.